### DISCUSSION TITLE
Retornar valor do campo de um combo no dynamic forms

### DISCUSSION THREAD

**Discussion by Smmartinez** [po-ui/po-angular#2033] [Tire suas dúvidas]
Opa pessoal, blz?

Preciso de uma ajuda, estou montando um Dynamic Forms e dentro dele tenho um combo para mostrar os dados de representante, estou fazendo assim:
  {
        property: 'codvend', label: 'Representante', gridColumns: 4, disabled: false,
        //options: this.getvendedor(), required: true,showRequired:true ,
        optionsService: '/apitlpp/v1/precadastro/vendedor',
        //searchService: "/apitlpp/v1/precadastro/vendedor",
        fieldLabel: 'descricao',
        fieldValue: 'codvend',
      },
para a inclusão de um novo processo, quando seleciono o combo ele dispara a API e traz os representantes, de forma correta.
mas quando vou em alterar e trago a informação da API esta ocorrendo o erro:
Os dados do vendedor que esta cadastrado não é apresentado no combo e quando clico nele ocorre o erro: " Cannot read property "codvend" of  "
Se eu falo esse processe sem ser por API, ou seja eu monto os dados na mão e passo no options quando retorna a api ele mostra o vendedor.

Alguém teria uma ideia ou uma solução para isso?

Esse é o retorno da API
{"items":[{"descricao":"CAPLAC COM.E REPRES.LTDA","codvend":"000075"}]}

obrigado!

**Comment by Smmartinez** [comment]
Alguém lê isso?

**Comment by anliben** [comment]
Bom Dia.

Anexe para mim um stackblitz com o ambiente que esta tentando fazer, versões do `po-ui`, angular, node

https://stackblitz.com/edit/po-ui

**Comment by Smmartinez** [comment]
Não consigo identificar por que ele não carrega a descrição do vendedor no campo, ele retorno no campo da api.
Estou montando assim:
 {
        property: 'codvend', label: 'Representante', gridColumns: 4, disabled: false,
        //options: this.getvendedor(), required: true,showRequired:true ,
        optionsService: '/apitlpp/precadastro/v1/vendedor',
        //searchService: "/apitlpp/v1/precadastro/vendedor",
        fieldLabel: 'label',
        fieldValue: 'value',
      },

![tela](https://github.com/po-ui/po-angular/assets/4532855/4c8830e8-71bd-4c6a-b57b-68a5828f9234)
